docs/maps/search.asciidoc

```diff
@@ -1,8 +1,13 @@
 [role="xpack"]
 [[maps-search]]
-== Searching your data
+== Search your data
```

Comment:
Can you just change to "Search"?

Reply:
Search by itself feels too sparse to me.  Search geographic data?

Reply:
sounds good.